src/index.js

```diff
@@ -5,6 +5,7 @@ export default function (state, actions, view, tagname) {
     actions._$ = function (x) { return x }
 
     return function (props) {
+        props.children = children;
```

Comment:
children is not in scope here. Needs to be added to the argument above

Reply:
Actually. Could you also make it:

```js
return function (props, children) {
  if (children.length > 0) props.children = children
```

... just in case someone who isn't using children of the component wants to use the name "children" as an internal state property.